.github/workflows/main.yml

```diff
@@ -38,6 +38,8 @@ jobs:
       - run: cd client && yarn install
       - run: cd client && yarn lint
       - run: cd client && yarn lint:test
+      - run: pipenv run test-server-up
```

Comment:
You could do this with yarn if you'd like, there is a script called test:server:up

Reply:
No need for that script anymore, since this way is more direct.